<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupPage</class>
 <widget class="QMainWindow" name="SignupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signup Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Copyright (c) DevSec Studio. All rights reserved.

MIT License

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the &quot;Software&quot;), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/

/*-----QWidget-----*/
QWidget
{
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #04456c, stop:0.33 #06183a, stop:0.67 #3d1437, stop:1 #231437);

	color: #ffffff;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: #292f45;
	color: #b9b9bb;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #f0742f;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 10px;

}


QPushButton::hover
{
	background-color: #fc7c11;
	color: #ffff;

}


QPushButton::pressed
{
	background-color: #ff6b35;
	color: #fff;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #292f45;
	color: #ffffff;
	border-style: solid;
	border-color: #000000;

}


QToolButton::hover
{
	background-color: #fc7c11;
	color: #000000;
	padding: 2px;
	border-radius: 15px;
	border-color: #fc7c11;

}


QToolButton::pressed
{
	background-color: #fc7c11;
	color: #000000;
	border-style: solid;
	border-width: 2px;

}


/*-----QLineEdit-----*/
QLineEdit{
	background-color: rgb(255, 255, 255);
	color: rgb(35, 35, 35);
	font-weight: bold;
	border-style: solid;
	border-width: 2px;
	border-top: 0px;
	border-left: 0px;
	border-right: 0px;
	border-color: #b9b9bb;
	padding: 10px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #b9b9bb;
	font-weight: bold;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #f0742f;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #1f2b2b;
    border: 1px solid #f0742f;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #f0742f;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="fullNamLlabel">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
 font-weight: bold;
background: None;
font-size: 20px;</string>
          </property>
          <property name="text">
           <string>FullName              </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fullNameLineEdit">
          <property name="styleSheet">
           <string notr="true">  position: relative;
  display: inline-block;
  height: 32px;
  width: 240px;
  margin: 0;
  padding: 0;
  background-color: #fff;
  border: none;
  border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
 font-weight: bold;
background: None;
font-size: 20px;</string>
          </property>
          <property name="text">
           <string>Username             </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="styleSheet">
           <string notr="true">  position: relative;
  display: inline-block;
  height: 32px;
  width: 240px;
  margin: 0;
  padding: 0;
  background-color: #fff;
  border: none;
  border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
 font-weight: bold;
background: None;
font-size: 20px;</string>
          </property>
          <property name="text">
           <string>Password               </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="styleSheet">
           <string notr="true">  position: relative;
  display: inline-block;
  height: 32px;
  width: 240px;
  margin: 0;
  padding: 0;
  background-color: #fff;
  border: none;
  border-radius: 10px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="confirmPasswordLabel">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
 font-weight: bold;
background: None;
font-size: 20px;</string>
          </property>
          <property name="text">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="confirmPassswordLineEdit">
          <property name="styleSheet">
           <string notr="true">  position: relative;
  display: inline-block;
  height: 32px;
  width: 240px;
  margin: 0;
  padding: 0;
  background-color: #fff;
  border: none;
  border-radius: 10px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="registerButton">
        <property name="font">
         <font>
          <family>Gilroy</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(254, 86, 106, 255), stop:1 rgba(213, 21, 53, 255));
border: none;
font-family: Gilroy,sans-serif;
</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegister">
    <property name="title">
     <string>Register</string>
    </property>
   </widget>
   <addaction name="menuRegister"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
